env0/resource_template.go

```diff
@@ -393,10 +393,22 @@ func templateCreatePayloadFromParameters(prefix string, d *schema.ResourceData)
 
 // Reads template and writes to the resource data.
 func templateRead(prefix string, template client.Template, d *schema.ResourceData) error {
+	pathPrefix := "path"
+	if prefix != "" {
+		pathPrefix = prefix + ".0.path"
+	}
+
+	path, pathOk := d.GetOk(pathPrefix)
+
 	if err := writeResourceDataEx(prefix, &template, d); err != nil {
 		return fmt.Errorf("schema resource data serialization failed: %v", err)
 	}
 
+	// https://github.com/env0/terraform-provider-env0/issues/699 - backend removes the "/".
+	if pathOk && path.(string) == "/"+template.Path {
```

Comment:
if diff is just in "/" revert to old path - this avoids a drift and keeps state intact with what the user entered.